testhelper/testAgentController.go

```diff
@@ -0,0 +1,50 @@
+package testhelper
```

Comment:
We already have a testutil package, could you move it over there?

Reply:
I know its not ideal, we originally had it in testutil but it created a lot of dependency cycle errors. There's a comment below in this package mentioning - `// Do not import other Telegraf packages as it causes dependency loops`

Reply:
In that case maybe it should have a very specific name like 'package testagentcontroller' so nobody will think it's a good place to add to in the future.

Reply:
Makes sense i've changed the name and added in the comment you suggested in your other comment